velox/type/Timestamp.h

```diff
@@ -284,6 +284,12 @@ struct Timestamp {
   /// Return whether the epoch second can be converted to a valid std::tm.
   static bool epochToUtc(int64_t seconds, std::tm& out);
 
+  /// Our own version of timegm to avoid expensive calls to __tz_convert.
+  ///
+  /// Return whether the conversion is successful.  This function is guaranteed
+  /// to be successful when the same input on timegm is successful.
+  static bool utcToEpoch(const std::tm& tm, int64_t& seconds);
```

Comment:
`utcToEpoch()` is a confusing name. Since you are converting a calendar date/time into unix epoch, maybe `calendarUtcToEpoch()` would be more descriptive?

Reply:
I am trying to keep consistent naming with `epochToUtc`.  Do we want to change that as well?

Reply:
if there are many callsites we could then change in a follow up PR, but the function name could be more descriptive. The jump from "utc" to "calendar time / tm" is not obvious.